src/scenes/CreateEvent/index.js

```diff
@@ -129,35 +131,125 @@ const messages = defineMessages({
+
+  if (bettingEndTime.unix() - bettingStartTime.unix() < TIME_GAP_MIN_SEC) {
+    errors.bettingEndTime = intl.formatMessage(messages.validBetEnd);
+  }
+
+  if (resultSettingStartTime.unix() < bettingEndTime.unix()) {
+    errors.resultSettingStartTime = intl.formatMessage(messages.validResultSetStart);
+  }
+
+  if (resultSettingEndTime.unix() - resultSettingStartTime.unix() < TIME_GAP_MIN_SEC) {
+    errors.resultSettingEndTime = intl.formatMessage(messages.validResultSetEnd);
+  }
+
+  return errors;
+};
+
 @withRouter
-class CreateEvent extends React.Component {
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state) => ({
+  initialValues: {
+    bettingStartTime: DEFAULT_PICKER_TIME_15_MIN,
+    bettingEndTime: DEFAULT_PICKER_TIME_45_MIN,
+    resultSettingStartTime: DEFAULT_PICKER_TIME_45_MIN,
+    resultSettingEndTime: DEFAULT_PICKER_TIME_75_MIN,
+    outcomes: ['', ''],
+  },
+  txReturn: state.Graphql.get('txReturn'),
+  walletAddresses: state.App.get('walletAddresses'),
+  createEventDialogVisible: state.App.get('createEventDialogVisible'),
+  eventEscrowAmount: state.Topic.get('eventEscrowAmount'),
+}), (dispatch) => ({
+  createTopicTx: (
+    name,
+    outcomes,
+    resultSetter,
+    bettingStartTime,
+    bettingEndTime,
+    resultSettingStartTime,
+    resultSettingEndTime,
+    escrowAmount,
+    creatorAddress,
+  ) => dispatch(graphqlActions.createTopicTx(
+    name,
+    outcomes,
+    resultSetter,
+    bettingStartTime,
+    bettingEndTime,
+    resultSettingStartTime,
+    resultSettingEndTime,
+    escrowAmount,
+    creatorAddress,
+  )),
+  toggleCreateEventDialog: (isVisible) => dispatch(appActions.toggleCreateEventDialog(isVisible)),
+  getInsightTotals: () => dispatch(appActions.getInsightTotals()),
+  changeFormFieldValue: (field, value) => dispatch(change(FORM_NAME, field, value)),
+}))
+@reduxForm({
+  form: FORM_NAME,
+  fields,
+  validate,
+})
+export default class CreateEvent extends Component {
   static propTypes = {
     classes: PropTypes.object.isRequired,
-    fields: PropTypes.array.isRequired,
     walletAddresses: PropTypes.array.isRequired,
     txReturn: PropTypes.object,
     createTopicTx: PropTypes.func,
     getInsightTotals: PropTypes.func,
-    history: PropTypes.object.isRequired,
     intl: intlShape.isRequired, // eslint-disable-line react/no-typos
     handleSubmit: PropTypes.func.isRequired,
     submitting: PropTypes.bool.isRequired,
     changeFormFieldValue: PropTypes.func.isRequired,
     toggleCreateEventDialog: PropTypes.func.isRequired,
     createEventDialogVisible: PropTypes.bool.isRequired,
     eventEscrowAmount: PropTypes.number,
-  };
+  }
 
   static defaultProps = {
     createTopicTx: undefined,
     txReturn: undefined,
     getInsightTotals: undefined,
     eventEscrowAmount: undefined,
-  };
+  }
 
   state = {
     selectAddressDialogVisibility: false,
-  };
+    error: {},
+    isValid: true,
```

Comment:
needs a more descriptive name specifying that it's for the non zero qtum check, like hasEnoughQTUM